src/test/specs/webdriver.js

```diff
@@ -0,0 +1,21 @@
+// import chai from 'chai';
+var { expect } = require('chai');
+
+describe('webdriver.io page', () => {
+    it('should have the right title', () => {
+        browser.url('/');
```

Comment:
I wonder if it's possible to point Webdriver at our storybook url and have it screenshot and diff stories similarly to our `loki` setup? We could use that in conjunction with diffing actual screens a user would see.

Reply:
Yeah I think that's definitely a the preferred path for our visual tests.

Reply:
It's pointing to one storybook url so far. If there is a way to generate a list of all the story point url's, then we should be able to pull it off

Reply:
I'm going to table this for the moment. It's still on the list for sure to tackle, but I haven't found a good way to get the list of stories and put it into a file that wdio can read. The getStorybook method that gets stories only shows data when loadStories command has been called, and that's only during the storybook setup so far. If I can get typescript further working with some of these changes, then perhaps we could just import the storybook config, and use the getStorybook command

Reply:
Well it turns out learning more about ts-node got me through this obstacle! I've generated all the stories in the following commit with a ts-node script.

https://github.com/cremalab/crema-app-web/pull/17/commits/6892961f823dba43844d356a057383c63e43dc94

While the stories are generated and I loop through them with wdio, I think we should be careful with how we document that pattern. Typically with wdio, you would write story specific scenarios such as clicking on a button or disabling animations for a particular component.